images/win/scripts/Installers/Post-Deployment.ps1

```diff
@@ -0,0 +1,19 @@
+# Create shells folder
+$shellPath = "C:\shells"
+$null = New-Item -Path $shellPath -ItemType Directory

```

Comment:
nitpick: may be use `<command> | Out-Null` instead of `$null = <command>`?

Reply:
Done.